lib/footerComponent.js

```diff
@@ -0,0 +1,18 @@
+import { createElement } from './elements';
+
+export function createFooterComponent() {
+    const footerElement = createElement(
+        'footer',
+        {
+            className: 'footer',
+        },
+        [
+            createElement(
+                'h1', {
```

Comment:
I think an h1 does not really make sense in the footer, you would rather find it in the header element. Maybe use a p tag here? :)

```suggestion
                'p', {
```

Reply:
you are absolutly right 
thanks lara 🎈